src/js/components/List/List.js

```diff
@@ -174,6 +174,18 @@ const List = React.forwardRef(
     // List items are likely selectable), active will be the
     // index of the item which is currently active.
     const [active, setActive] = useState();
+    const activeItemRef = useRef(null);
+
+    const setActiveItemRef = (element) => {
+      if (element) {
+        element.scrollIntoView({
```

Comment:
Can we only scroll the element into view if we are navigating with a keyboard?
You can use `useKeyboard` from 'src/js/utils' to help determine if someone is navigating with a keyboard or if they are using a mouse.

Also can we check if the list item is fully visible or not before triggering the scrolIntoView function so that we don't call it if we don't need to